webapp/app/js/perf.js

```diff
@@ -6,25 +6,273 @@
 
 var perf = angular.module("perf", ['ui.router', 'ui.bootstrap', 'treeherder']);
 
-perf.factory('getSeriesSummary', [ function() {
-  return function(signature, signatureProps, optionCollectionMap) {
-    var platform = signatureProps.machine_platform + " " +
-      signatureProps.machine_architecture;
-    var e10s = (signatureProps.job_group_symbol === "T-e10s");
-    var testName = signatureProps.test;
-    var subtestSignatures;
-    if (testName === undefined) {
-      testName = "summary";
-      subtestSignatures = signatureProps.subtest_signatures;
-    }
-    var name = signatureProps.suite + " " + testName;
-    var options = [ optionCollectionMap[signatureProps.option_collection_hash] ];
-    if (e10s) {
-      options.push("e10s");
+perf.factory('phSeries', ['$http', 'thServiceDomain', function($http, thServiceDomain) {
```

Comment:
For whatever reason, the convention seems to be to capitalize factories. I.e. phSeries should be PhSeries.